content/docs/1.9.0/advanced-resources/data-cleanup/orphaned-instance-cleanup.md

```diff
@@ -0,0 +1,252 @@
+---
+title: Orphaned Instance Cleanup
+weight: 4
+---
+
+Longhorn can identify and clean up orphaned instance processes on each node.
+
+## Orphaned Runtime Instance
+
+When a network outage affects a Longhorn node, it may leave behind engine or replica runtime instances that are no longer tracked by the Longhorn system. The corresponding engine and replica CRs might be removed or rescheduled to another node during the outage. When the node comes back, the Longhorn system no longer tracks the corresponding runtime resources. These runtime resources, including the processes, are called orphaned instances. Orphaned instances continue to consume CPU and memory.
```

Comment:
```suggestion
When a network outage affects a Longhorn node, it may leave behind engine or replica runtime instances that are no longer tracked by the Longhorn system. The corresponding engine and replica CRs might be removed or rescheduled to another node during the outage. When the node comes back, the Longhorn system no longer tracks the corresponding runtime instances. These runtime instances, such as the engine and replica processes for v1 volumes, are called orphaned instances. Orphaned instances continue to consume CPU and memory.
```